src/Utility/Text.php

```diff
@@ -852,4 +859,60 @@ public static function parseFileSize($size, $default = false)
         }
         throw new InvalidArgumentException('No unit type.');
     }
+
+    /**
+     * Transliterate string.
+     *
+     * @param string $string String to transliterate.
+     * @param string|null $transliteratorId Transliterator identifer. If null
+     *   Text::$defaultTransliteratorId will be used.
+     * @return string
+     * @see http://php.net/manual/en/transliterator.transliterate.php
+     */
+    public static function transliterate($string, $transliteratorId = null)
+    {
+        $transliteratorId = $transliteratorId ?: static::$defaultTransliteratorId;
+        return transliterator_transliterate($transliteratorId, $string);
```

Comment:
@admad is there a particular reason why you used the "procedural" type instead of the object oriented one (``Transliterator::transliterate``)?
It just reminds me of the good old date functions, which we today replaced with the DateTime etc. objects...